### PR TITLE
issue-983 accessible closed captions + issue-854 fullscreen fixes

### PR DESCRIPTION
**Pull request by oliverfoster**
* added patch for closed captions buttons
* added patch for fullscreen

### CODE REVIEW

**Comment by hrajotia** [comment]
+1

**Comment by tomgreenfield** [comment]
Hate to be the pedant but would prefer <code>_allowFull**S**creen</code>. 

Nice use of override.

+1

**Comment by moloko** [comment]
going to be testing this in a project today/tomorrow, will let you know how I get on with that

**Comment by moloko** [comment]
OK it ain't looking so pretty in IE9 - obviously because it's trying to do 'fake fullscreen' due to IE9's lack of support for full screen video

I suggest we just disable this for IE9

**Comment by oliverfoster** [comment]
ewww

File: js/adapt-contrib-media.js [hunk 1]
```diff
@@ -34,7 +36,10 @@ define(function(require) {
             if(modelOptions.features === undefined) {
                 modelOptions.features = ['playpause','progress','current','duration'];
                 if (this.model.get('_useClosedCaptions')) {
-                    modelOptions.features = ['playpause','progress','tracks','current','duration'];
+                    modelOptions.features.unshift('tracks');
+                }
+                if (this.model.get("_allowFullscreen")) {
```
Comment (moloko) [inline js/adapt-contrib-media.js]: how about
```
if (this.model.get("_allowFullscreen") && !$('html').is('.ie9')) {
```

**Comment by moloko** [comment]
I agree with @tomgreenfield , `_allowFullScreen` is better

**Comment by tomgreenfield** [comment]
+1

**Comment by hrajotia** [comment]
+1

**Comment by moloko** [comment]
+1

**Pull request merged**